code/modules/projectiles/projectile/magic.dm

```diff
@@ -424,3 +424,22 @@
 	exp_light = -1
 	exp_flash = 4
 	exp_fire= 5
+
+/obj/item/projectile/magic/aoe/shockwave
+	name = "kinetic shockwave"
+	icon_state = "shockwave"
+	damage = 15
+	damage_type = BRUTE
+	nodamage = 0
```

Comment:
This will do nothing if there are no living targets. It would be pretty awkward to ground pound and do nothing because there are no people around you to believe in you. I don't have a good solution for this short of changing the spell effect into something already existing (repulse) or making a new spell that throws projectiles regardless of targets.